src/components/Text/Text.module.css

```diff
@@ -92,3 +92,156 @@
 .text--2x-spacing {
   margin-bottom: var(--eds-size-2);
 }
+
+/**
+ * 1) A passage of text, including various components (i.e. article, blog post)
+ */
+.text-passage {
+  /**
+	 * Paragraph within text passage
+	 */
+  p {
+    margin-bottom: var(--eds-size-3);
+  }
+
+  /**
+   * First child of a text passage
+   * 1) Zero out top margin to avoid unintended
+   *    gaps when leading with a heading or other
+   *    element that has a top margin
+   */
+  > :first-child {
+    margin-top: 0; /* 1 */
+  }
+
+  /**
+   * Last child of a text passage
+   * 1) Zero out bottom margin to allow the container
+   *    element to control the bottom margin
+   */
+  > :last-child {
+    margin-bottom: 0;
+  }
+
+  /**
+	 * Link within the text passage
+	 */
+  a {
+    @mixin textLink;
+  }
+
+  /**
+	  * First-level heading within text passage
+	  */
+  h1 {
+    @mixin eds-theme-typography-headline-lg;
+  }
+
+  /**
+	 * Second-level heading within text passage
+	 */
+  h2 {
+    @mixin eds-theme-typography-headline-md;
+    margin-top: var(--eds-size-5);
+  }
+
+  /**
+	 * Third-level heading within text passage
+	 */
+  h3 {
+    @mixin eds-theme-typography-headline-sm;
+    margin-top: var(--eds-size-5);
+  }
+
+  /**
+	 * Fourth-level heading within text passage
+	 */
+  h4 {
+    @mixin eds-theme-typography-title-md;
+    margin-top: var(--eds-size-5);
+  }
+
+  /**
+	 * Fifth-level heading within text passage
+	 */
+  h5 {
+    @mixin eds-theme-typography-title-sm;
+    margin-top: var(--eds-size-5);
+  }
+
+  /**
+	 * Sixth-level heading within text passage
+	 */
+  h6 {
+    @mixin eds-theme-typography-body-text-sm-bold;
+    margin-top: var(--eds-size-5);
+  }
```

Comment:
These mixins are not the same as the mixins used for `h1` - `h6` in the `Heading` component, which mixins should be used?

Reply:
The mixins are being updated in https://github.com/chanzuckerberg/edu-design-system/pull/1075/files#r881562418 so the ones here and in `Heading` will produce the same styles.

Reply:
Thanks!